book.rb

```diff
@@ -0,0 +1,13 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(book)
+    @rentals.push(book)
+  end
```

Comment:
- [ ] The `@rentals` should contain a list of rentals not a list of books. Kindly implement this in this method (its implementation should be similar to the `add_rental` method in the `Person` class, also please be mindful of what parameters to pass when creating a new rental here and their order)